graveler/committed/sstable/serialization.go

```diff
@@ -0,0 +1,10 @@
+package sstable
+
+import "github.com/treeverse/lakefs/graveler"
+
+// serializer is a placeholder interface for committed way of
+// serializing the graveler.Value
+type serializer interface {
+	SerializeValue(inVal graveler.Value) ([]byte, error)
```

Comment:
:spiral_notepad:  See #1067.  The plan is to move this _outside_ of this directory, `committed` doesn't care.  (No action required here and now, of course!)

Reply:
Yap - that's my next PR.